ui/Makefile

```diff
@@ -11,7 +11,8 @@ SOURCES += $(shell find apps -type d \( -name node_modules \) -prune -o -print)
 # tracking package.json files of monorepo packages is the easiest way to discover such situations
 PACKAGE_JSON_FILES := $(shell find . -type d \( -name node_modules \) -prune -false -o -name package.json)
 
-export REACT_APP_ROX_PRODUCT_BRANDING := $(shell $(MAKE) --quiet --no-print-directory -C .. product-branding)
+ROX_PRODUCT_BRANDING ?= $(shell $(MAKE) --quiet --no-print-directory -C .. product-branding)
+export REACT_APP_ROX_PRODUCT_BRANDING := $(ROX_PRODUCT_BRANDING)
```

Comment:
@vjwilson

This makes `REACT_APP_ROX_PRODUCT_BRANDING` configurable via `ROX_PRODUCT_BRANDING` environment variable when the latter one is externally set (`?=` assignment).
Basically, both variables have to be set to the same values and `ROX_PRODUCT_BRANDING` is the one that drives everything.

I made the change because I noticed that the old code which calls Makefile in the repo root... requires Golang and other goodies to be installed. We had no issues with that presumably because UI engineers have Golang (God, I hope not because I forced them with my original dumb change) and downstream builder image also has it.

A builder image in RHTAP doesn't have Golang and I saw problems during the build.

The PR is still draft but I'd appreciate your or someone from UI team eyes on it when it's ready.

Reply:
^ @alwayshooin @sachaudh @pedrottimark @dvail @bradr5

Reply:
Factored this in a separate PR https://github.com/stackrox/stackrox/pull/9620